app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -255,18 +288,19 @@ class WeightedQueryStrategy
       def initialize(user: nil, number_of_articles: 50, page: 1, tag: nil, **config)
         @user = user
         @number_of_articles = number_of_articles.to_i
-        @page = page.to_i
+        @page = (page || 1).to_i
         # TODO: The tag parameter is vestigial, there's no logic around this value.
         @tag = tag
         @default_user_experience_level = config.fetch(:default_user_experience_level) { DEFAULT_USER_EXPERIENCE_LEVEL }
         @negative_reaction_threshold = config.fetch(:negative_reaction_threshold, DEFAULT_NEGATIVE_REACTION_THRESHOLD)
         @positive_reaction_threshold = config.fetch(:positive_reaction_threshold, DEFAULT_POSITIVE_REACTION_THRESHOLD)
-        @oldest_published_at = determine_oldest_published_at(
-          user: @user,
-          most_number_of_days_to_consider: config.fetch(:most_number_of_days_to_consider, 31),
-        )
         @scoring_configs = config.fetch(:scoring_configs) { default_scoring_configs }
         configure!(scoring_configs: @scoring_configs)
+
+        @oldest_published_at = self.class.oldest_published_at_to_consider_for(
```

Comment:
`self.class` here is used to support specializing this method in future subclasses?

Reply:
I think it's primarily a class method because it doesn't need internal state and is directly called in `app/services/articles/feeds/large_forem_experimental.rb:104` as part of this PR. I find it slightly unintutive that one strategy calls a class method of the other and that class itself also calls its own class method. Maybe this is a good indication that this should be something else entirely, maybe something like `FeedQueries::OldestPublishedAtToConsiderFor` or something similar.

Reply:
My original preference was `Articles::Feeds.oldest_published_at_to_consider_for` but we don't have a `app/services/articles/feeds.rb` so I "cheated."  However, I'm going to uncheat and make a proper `Article::Feed` or `Article::Feeds` module.

I'll have more in the a future commit, as well as commentary/discussion on why the module.